### DISCUSSION TITLE
When it's ready, it's going to be incredible.

### DISCUSSION THREAD

**Discussion by andersonantunes26** [Monika-After-Story/MonikaModDev#8841] [appreciation]
I thank all hard work but I hope for this year ...
1-a piano on the floating island or Monika's drawing room
![yy7yjgosmln11](https://user-images.githubusercontent.com/11380696/53127167-39763680-3538-11e9-8e32-8d7599770318.jpg)
2- different hairstyles
![34998375-e8e48e22-faac-11e7-8388-e5f79e22c9f7](https://user-images.githubusercontent.com/11380696/53127254-69253e80-3538-11e9-8c47-32434150077c.png)
3-a little more movements like these and pokes na monika
![hqdefault](https://user-images.githubusercontent.com/11380696/53127447-d638d400-3538-11e9-9cbd-a2b9b20ad941.jpg)
https://i.redd.it/41q5ylgvcw301.gif
https://i.kym-cdn.com/photos/images/newsfeed/001/306/555/77b.gif

Thank you for all the work well done until this year continue like this.

**Comment by kaido1224** [comment]
Hair styles aren't likely

**Comment by softombre** [comment]
Why did you post this in like 3 issues
